src/app/public/modules/affix/affix.directive.spec.ts

```diff
@@ -478,11 +559,165 @@ describe('Affix directive', () => {
   });
 
   it('should be accessible', async(() => {
-    componentInstance.enableScrollableParent = true;
+    componentInstance.enableOverflowParent = true;
     componentInstance.scrollTargetToTop();
     fixture.detectChanges();
     fixture.whenStable().then(() => {
       expect(fixture.nativeElement).toBeAccessible();
     });
   }));
+
+  it('should work when the base element is larger than the affixed element', () => {
```

Comment:
Would you mind expanding on "it should work..." ? What's the expected result?

Reply:
Yep, that's horribly undescriptive.